plugins/terminal.koplugin/main.lua

```diff
@@ -1,58 +1,378 @@
+local ButtonDialog = require("ui/widget/buttondialog")
+local CenterContainer = require("ui/widget/container/centercontainer")
 local DataStorage = require("datastorage")
 local Font = require("ui/font")
 local InfoMessage = require("ui/widget/infomessage")
 local InputDialog = require("ui/widget/inputdialog")
+local LuaSettings = require("luasettings")
+local Menu = require("ui/widget/menu")
 local TextViewer = require("ui/widget/textviewer")
 local Trapper = require("ui/trapper")
 local UIManager = require("ui/uimanager")
 local WidgetContainer = require("ui/widget/container/widgetcontainer")
 local logger = require("logger")
 local util = require("ffi/util")
 local _ = require("gettext")
+local lfs = require("libs/libkoreader-lfs")
 local Screen = require("device").screen
 
-local Terminal = WidgetContainer:new{
+local Terminal = WidgetContainer:new {
```

Comment:
No need to change that.
```suggestion
local Terminal = WidgetContainer:new{
```